waspc/data/Generator/templates/server/src/config.js

```diff
@@ -35,5 +34,5 @@ const config = {
   }
 }
 
-const resolvedConfig = _.merge(config.all, config[env])
+const resolvedConfig = {...config.all, ...config[env]}
```

Comment:
There is a difference here though -> `_.merge` does the merge recursively, while spread operator will merge only at the top level.
So if we have
```js
let foo = { a: { b: 3 } }
let bar = { a: { c: 4 } }
```

`_.merge(foo, bar)` will give `{ a: { b: 3, c: 4 } }` while `{...foo, ...bar}` will give `{ a: { c: 4 } }`.

Maybe we ok with such behaviour, but I just wanted to warn it is a change indeed. Maybe it is safer to go with recursive merge here.

Reply:
Yeah, I had thought about that and was trying to find a non-library recursive solution (I didn't even remember I just did the quick and dirty merge to remove lodash), and was tinkering with my own/others' recursive solutions. This def would not go in like this, but I appreciate the note here. 👍🏻 

In general, nothing in this Draft PR would be considered mergable just FYI. Only testing dep upgrades/removals atm. Thanks!